eks_infra/upgrade_cluster.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+#/ Usage:
+#/ export AWS_REGION=<AWS-Region>
+#/ export EKS_CLUSTER_MANAGER_ROLE=<ARN-of-IAM-role>
+#/ ./upgrade.sh eks_cluster_name eks_version cluster_autoscalar_image_version
```

Comment:
nit: I think this is the wrong file name, right? should be `upgrade_cluster.sh`?

Reply:
Ack, corrected